### ISSUE TITLE
Nemesis Gaining Intangible Crash

### ISSUE THREAD

**Issue by timeracers** [daviscook477/BaseMod#15]
At the end of the first turn of the Nemesis fight he gains Intangible which silently crashes the game.

**Comment by daviscook477** [comment]
Yeah I don't know why. If you have any ideas I'd love to hear them

**Comment by daviscook477** [comment]
It says that there's an index out of bounds exception when accessing the descriptions but I can't see where it's going wrong

**Comment by daviscook477** [comment]
Okay I figured out the problem @timeracers . It was an issue with my maven build process and not actually any of the code. Maven leaves some artifacts from every time it builds and for some reason doesn't remove them when files change. So when BaseMod used to directly modify the IntangiblePower there was a file that referenced the 3rd string in the description for Intangible and now that the game changed to only have 1 string in the description for Intangible it crashes with an indexofoutboundsexception. I figured this out by building on my laptop which never had that old version of BaseMod and successfully fought Nemesis without a crash. I'll resolve that build artifact and push a new build with that issue fixed
